packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -185,6 +188,11 @@ class FlutterCommandRunner extends CommandRunner<void> {
   Future<void> runCommand(ArgResults topLevelResults) async {
     final Map<Type, Object?> contextOverrides = <Type, Object?>{};
 
+    // If the disable-telemetry flag has been passed, return out
```

Comment:
Why are we early exiting here?

Reply:
ie. the first line the code block above is probably all the user cares about